bot/exts/halloween/timeleft.py

```diff
@@ -35,16 +38,20 @@ async def timeleft(self, ctx: commands.Context) -> None:
         Calculates the time left until the end of Hacktober.
 
         Whilst in October, displays the days, hours and minutes left.
-        Only displays the days left until the beginning and end whilst in a different month
+        Only displays the days left until the beginning and end whilst in a different month.
+
+        This factors in that Hacktoberfest starts when it is October anywhere in the world
+        and ends with the same rules. It treats the start as UTC+14:00 and the end as
+        UTC-12.
         """
         now, end, start = self.load_date()
         diff = end - now
         days, seconds = diff.days, diff.seconds
-        if self.in_october():
+        if self.in_hacktober():
             minutes = seconds // 60
             hours, minutes = divmod(minutes, 60)
-            await ctx.send(f"There is currently only {days} days, {hours} hours and {minutes}"
-                           "minutes left until the end of Hacktober.")
+            await ctx.send(f"There are {days} days, {hours} hours and {minutes}"
+                           " minutes left until the end of Hacktober.")
```

Comment:
```suggestion
            await ctx.send(
        f"There are {days} days, {hours} hours and {minutes}"
        f" minutes left until the end of Hacktober."
    )
```

Reply:
Addressed in 60a024c818c405822f13046ff93805837bfc591e, formatting on this suggestion seemed a bit off.